src/react/TitleProvider.tsx

```diff
@@ -14,6 +16,20 @@ const ticksToMs = (ticks: AnimationTimes) => {
   return ticks
 }
 
+const getComponent = (input: string | any) => {
```

Comment:
```suggestion
const getComponent = (input: string | Record<string, any>) => {
```

Reply:
👍 makes sense. I'm not that well versed in TypeScript so I don't really know the appropriate types at times 😅